usaspending_api/api_contracts/contracts/v2/disaster/agency/count.md

```diff
@@ -0,0 +1,100 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Count of Agencies receiving disaster/emergency funding [/api/v2/disaster/agency/count/]
```

Comment:
If you go to make more changes would be good to capitalize this header like the other contracts for consistency.

Reply:
I think I addressed this for all files. Need to verify